ODF-community.french.md

```diff
@@ -0,0 +1,526 @@
+Decision Framework" en anglais) a été créé par l'équipe Red Hat People et est
+disponible sous licence Creative Commons Attribution-ShareAlike 4.0
+International License ([CC BY-SA
+4.0](https://creativecommons.org/licenses/by-sa/4.0/)). Red Hat et le logo
+Shadowman sont des marques déposées de Red Hat, Inc. et enregistrée dans
+d'autres pays. La marque Red Hat doit être retirée de toute version modifiée.
+
+Notes
+=========
+
+* Cette version markdown des diapositives est publiée pour faciliter la
+  collaboration et le suivi des changements.
+* La numérotation des pages correspond aux fichiers LibreOffice (.odp) et PDF
+  dans le dépot.
+
+Contenu
+===========================================================================
+
+* [Vue d'ensemble](#overview)
+  * [Qu'est-ce qu'une décision ouverte ?](#what-is-an-open-decision)
+  * [Les décisions ouvertes sont issues des principes open source](#open-decisions-open-principles)
+  * [Comment les principes open source conduisent à de meilleures décisions](#how-open-source-principles)
+  * [On ne peut pas plaire à tout le monde](#cant-please-everyone)
+* [Cadre de Décision Ouverte](#framework)
+  * [Phase ou activité: Conceptualiser, Définir, Imaginer](#ideation)
+  * [Phase ou activité: Planifier, Rechercher](#plan-research)
+  * [Phase ou activité: Concevoir, Developper, Tester](#design-develop-test)
+  * [Phase ou activité: Lancer, Deployer, Cloturer](#launch)
+* [Resources](#resources)
+* [Annexes](#appendix)
+  * [Histoire: D'où vient le Cadre de Décision Ouvert](#history)
+  * [Pourquoi ce cadre existe](#why-framework)
+
+<br/>
+
+Vue d'ensemble<a name="overview"></a>
+===========================================================================
+
+_page 2 sur 15_
+
+**Qu'est-ce c'est ?**
+  - Une approche flexible et ouverte pour prendre des décisions en entreprise et
+    mener des projets.
+
+**Quand l'utiliser ?**
+
+Pour des décisions et des projets qui :
+  - impactent notre culture ou
+  - affectent des collaborateurs au-delà de votre équipe immédiate
+
+**Comment l'utiliser ?**
+  - Intégrer par étapes le Cadre de Décision Ouverte dans votre projet ou dans
+    votre processus de prise de décisions.
+
+<br/>
+
+Qu'est-ce qu'une décision ouverte ? <a name="what-is-an-open-decision"></a>
+-------------------------
+
+_page 3 sur 15_
+
+<table>
+<tr>
+<td>Transparente</td>
+<td>Inclusive</td>
+<td>Centrée sur l'usager</td>
+</tr>
+<tr>
+<td>Expliquer qui prend la décision, quels problèmes vous essayez de régler, les
+pré-requis et les contraintes en jeu et le processus que vous aller suivre.</td>
+<td>Solliciter le retour d'autres personnes et collaborer tout au long du
+processus de décision. Rechercher des points de vue variés, notamment ceux des
+détracteurs potentiels.</td>
+<td>Considérer les personnes comme des usagers ayant dans des besoins
+contradictoires et des priorités. Quand une décision aide certains usagers
```

Comment:
Sounds good to me. The original versions says `Think of people as customers with competing needs and priorities.`